deployments/helm-chart/templates/rbac.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.rbac.create }}
 kind: ClusterRole
-apiVersion: rbac.authorization.k8s.io/v1beta1
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Can you make the same change?:
https://github.com/nginxinc/kubernetes-ingress/blob/gh-actions-edge/deployments/rbac/rbac.yaml#L2

Reply:
Done, thanks!